libs/server/repository/infrastructure/src/lib/repositories/repository-repository.adapter.ts

```diff
@@ -3,13 +3,12 @@ import { InjectRepository } from '@nestjs/typeorm';
 import { RepositoryEntity } from '@pimp-my-pr/server/repository/core/domain';
 import { RepositoryRepository } from '@pimp-my-pr/server/repository/core/domain-services';
 import { Repository } from 'typeorm';
-import { GithubRepositoryRepository } from '../github/repositories/github-repository.repository';
 
 @Injectable()
 export class RepositoryRepositoryAdapter extends RepositoryRepository {
   constructor(
     @InjectRepository(RepositoryEntity) private typeOrmRepository: Repository<RepositoryEntity>,
-    private githubRepository: GithubRepositoryRepository
+    private repository: RepositoryRepository
```

Comment:
RepositoryRepositoryAdapter extends RepositoryRepository. I think you meant something like `remoteRepository: RemoteRepositoryRepository`